app/i18n/en/videoDebate.json

```
@@ -18,7 +18,10 @@
     "flagForm": {
         "title": "Why do you want to flag this comment?",
         "xAvailable": "{{count}} available",
-        "xAvailable_plural": "{{count}} available"
+        "xAvailable_plural": "{{count}} available",
+        "warningMessage1": "If one of your flags is considered overrated, you will lose reputation.",
```

`overrated` seems a little too light

```suggestion
        "warningMessage1": "If one of your flags is considered abusive, you will lose reputation.",
```